dm/tests/lightning_mode/run.sh

```diff
@@ -73,10 +73,10 @@ function run() {
 	run_tidb_server 4000 $TIDB_PASSWORD
 }
 
-cleanup_data lightning_mode
-# also cleanup dm processes in case of last run failed
-cleanup_process $*
-run $*
-cleanup_process $*
+#cleanup_data lightning_mode
```

Comment:
should such changes be reverted？

Reply:
it seems these two dm integration tests need be fixed